pyproject.toml

```diff
@@ -3,9 +3,10 @@ build-backend = "setuptools.build_meta"
 requires = ["setuptools>=50.0.0", "wheel"]
 
 [tool.pytest.ini_options]
-addopts = "-r s --cov=./src/r5py --cov-append --cov-report term-missing --cov-report xml"
+addopts = "-p no:faulthandler -r s --cov=./src/r5py --cov-append --cov-report term-missing --cov-report xml"
```

Comment:
Is `no:faulthandler` used to ignore the segfault warnings, or is it something else?

Reply:
It's needed because of how jpype is designed (replacing what was in conftest.py). If I understand correctly, the jpype superimposes the python faulthandler on top of/instead of (not sure) the JVM's. When pytest adds its own still on top of those, things stop working. 

https://github.com/jpype-project/jpype/issues/842